### ISSUE TITLE
Weird jittery behavior when `background()` is not called in `draw()` while using the skia renderer.

### ISSUE THREAD

**Issue by procub3r** [p5py/p5#425]
**Describe the bug**
When `background()` is not called in `draw()`, there is a weird jittery behavior that is observed while the graphics are being rendered using the skia renderer.

**To Reproduce**
Run an example program:
```python
from p5 import *

def size():
    pass

def draw():
    # Do not call background() here.
    circle(mouse_x, mouse_y, 50)

run(renderer='skia')
```

**Expected behavior**
Smooth rendering without any weird jittery behavior.

**Screen capture**

https://user-images.githubusercontent.com/54534811/220842191-4b648854-e853-4ead-a56d-1c20671e64da.mov

**System information:**
 - p5 release: Current master branch 8f4777b
 - Python version: 3.10.10
 - Operating system: Windows 11

**Additional context**
https://github.com/p5py/p5/pull/423#issuecomment-1440565966

**Comment by tushar5526** [comment]
This seems to be related to some bug with swapping of frame buffer and this was not the case earlier. I think a good strategy would be to go a few commits back and identify for the buggy commit

**Comment by tushar5526** [comment]
https://github.com/p5py/p5/commit/a0f1a046c8d81af106b1937a7ecfca1fc85a1879 -- This is the commit that produced this bug. Things are working fine before this commit. Would you like to do a fix for this ?

**Comment by tushar5526** [comment]
We only need to disable this line - https://github.com/p5py/p5/commit/6b079cf3fb19f691fdd254e3bb3c82c38905cdeb#diff-7a18b014da674a8330d705e9c488589c23a673a0062af59d7d1e1270181f673aL80 and test it on windows.

**Comment by procub3r** [comment]
Cool! I'll get in on this.

**Comment by procub3r** [comment]
I'm slightly confused :sweat_smile: 
What do you mean by disabling that line? That line doesn't exist in the master branch anymore.

Adding back all three of those lines seems to work (yet to test on windows though).

**Comment by procub3r** [comment]
Adding back all three of those lines seems to error out on windows. This is the error that I get:
```
Traceback (most recent call last):
  File "c:\dev\p5dev\test\main.py", line 9, in <module>
    run(renderer='skia')
  File "C:\Users\procub3r\AppData\Local\Programs\Python\Python310\lib\site-packages\p5\sketch\userspace.py", line 231, in run
    p5.sketch.start()
  File "C:\Users\procub3r\AppData\Local\Programs\Python\Python310\lib\site-packages\p5\sketch\Skia2DRenderer\base.py", line 159, in start
    self.window = self.glfw_window()
  File "C:\Users\procub3r\AppData\Local\Programs\Python\Python310\lib\site-packages\p5\sketch\Skia2DRenderer\base.py", line 80, in glfw_window
    GL.glEnable(GL.GL_MULTISAMPLE)
  File "C:\Users\procub3r\AppData\Local\Programs\Python\Python310\lib\site-packages\OpenGL\platform\baseplatform.py", line
415, in __call__
    return self( *args, **named )
  File "src/errorchecker.pyx", line 58, in OpenGL_accelerate.errorchecker._ErrorChecker.glCheckError
OpenGL.error.GLError: GLError(
        err = 1282,
        description = b'invalid operation',
        baseOperation = glEnable,
        cArguments = (GL_MULTISAMPLE,)
)
```
Adding back just the first two lines and commenting out `GL.glEnable(GL.GL_MULTISAMPLE)` doesn't produce an error but it doesn't fix the issue either. The jittery behavior persists.

**Comment by procub3r** [comment]
My initial thought was that maybe the buffers were being swapped more than once in a single frame due to some bug. But I've just tested it and that doesn't seem to be the case.

Unrelated note: I don't quite understand the role `GL.glEnable(GL.GL_MULTISAMPLE)` plays here.
I mean for some reason, when I add that line in, the jittery behavior stops on linux and it works smoothly. But it simply errors out on windows.

**Comment by tushar5526** [comment]
Sorry for all the confusion. I removed those 3 lines to remove the error that you were facing. Those lines are some configurations for GLFW related to anti-aliasing etc. you can read about them in the glfw docs. 

> I mean for some reason, when I add that line in, the jittery behavior stops on linux and it works smoothly. But it simply errors out on windows.

This is what we have to fix for. Research why its throwing error on windows and workarounds for it.

**Comment by tushar5526** [comment]
https://github.com/p5py/p5/issues/399 - more context for you.

**Comment by procub3r** [comment]
Thanks for the clarification! I know what to do now :D

**Comment by tushar5526** [comment]
Closed by #446
